src/ensembl/production/metadata/grpc/utils.py

```diff
@@ -415,3 +417,32 @@ def get_genome_uuid_by_tag(db_conn, genome_tag):
             {"genome_uuid": genome_uuid_result[0].Genome.genome_uuid}
         )
     return msg_factory.create_genome_uuid()
+
+
+def get_ftp_links(db_conn, genome_uuid, dataset_type, release_version):
+
+    # Request is sending an empty string '' instead of None when
+    # an input parameter is not supplied by the user
+    if not genome_uuid:
+        return msg_factory.create_paths()
+    if not dataset_type:
```

Comment:
I don't think it's necessary to have the `dataset_type` and `release_version` conditions because by default `get_public_path()` [sets them by default](https://github.com/Ensembl/ensembl-metadata-api/blob/dbfeeedc069811b049374efdd92c75446f88a029/src/ensembl/production/metadata/api/models/genome.py#L41):
```
def get_public_path(self, type='all', release=None):
   ...
```

Reply:
As discussed in person, this is necessary due to the changed behaviour where empty strings are being passed instead of None.